lib/charms/layer/docker_registry.py

```diff
@@ -99,6 +99,14 @@ def configure_registry():
             'container': charm_config.get('storage-swift-container', ''),
             'tenant': charm_config.get('storage-swift-tenant', ''),
         }
+
+        # Openstack Domain settings (https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/swift.md)
+        # domain or domainid - will always default to domain if both are specified.
+        for key in ['domain', 'domainid']:
+            val = charm_config.get('storage-swift-{}'.format(key), '')
+            if val is not '':
+                storage['swift'].update({key: val})
+
```

Comment:
simplified by only supporting `domain`, i.e. drop the `for` loop.

Reply:
Hey @kwmonroe it's not marked as outdated, can you mark it possibly?